examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -38,8 +38,8 @@
 fig.histogram(
     projection="X10c/2c",
     frame=["Wsrt", "xf1", "y+lCounts"],
-    # Give the same value for ymin and ymax to have ymin and ymax
-    # calculated automatically
+    # Give the same value for ymin and ymax to have ymin set to zero
```

Comment:
Looking at the GMT source code (https://github.com/GenericMappingTools/gmt/blob/fa30eeb00a2205c010f7db61129d060360f9a67c/src/pshistogram.c#L1256), *ymin* is not zero for logarithm projections. I think the old explanation is more general and looks better.

Reply:
Ah. I only considered linear projections. Then we can probably close this PR. I also reverted this change in the tutorial in commit https://github.com/GenericMappingTools/pygmt/pull/2445/commits/4e8610af8b9f83a7731e67a9c5c02b1d5112aeed.